lua/pulsar_lib/core/sql/sv_sql.lua

```diff
@@ -106,6 +138,61 @@ function SQL:RawQuery(query, onSuccess, onError)
 	end
 end
 
+function SQL:PreparedQuery(query, values, onSuccess, onError)
+	onSuiccess = onSuccess or emptyFunction
```

Comment:
`onSuiccess` should be `onSuccess`

Reply:
Didn't noticed it first, but it will get fixed ^^